guides/v2.2/cloud/project/project-conf-files_services-mysql.md

```diff
@@ -10,21 +10,19 @@ The `mysql` service provides persistent data storage based on [MariaDB](https://
 
 {% include cloud/service-config-integration-starter.md %}
 
-#### To access the MariaDB database directly:
+Accessing the MariaDB database directly requires you to use a SSH to log in to the remote server, and connect to the database with the following credentials:
```

Comment:
I see "a SSH" again. It doesn't sound right to me, but maybe you had something else in mind–an SSH connection, or "use SSH"

Reply:
In my head it is `a secure shell`...people may pronounce the `s` in their head instead of `secure`. It doesn't matter to me.